voluncheer/profiles/views/activate_email.py

```diff
@@ -31,7 +29,7 @@ def activateEmail(request, user, to_email):
         send_mail(
             subject,
             message,
-            "admin@admin.com",
+            os.getenv("EMAIL"),
```

Comment:
Is this the "from" email? Can you update this one to be more descriptive? Here and in `profile.py`.

Reply:
This was updated in later commits.